forc-plugins/forc-publish/src/main.rs

```diff
@@ -29,6 +29,7 @@ async fn main() {
     init_tracing_subscriber(TracingSubscriberOptions::default());
 
     if let Err(err) = run().await {
+        println!();
         println_error(&format!("{err}"));
```

Comment:
note: this may result in empty line to stdout; and error to stderr below; if this is not intended; and you only want stderr output (no stdout) - you may want to:

```suggestion
        println_error(&format!("\n{err}"));
```
or
```suggestion
        eprintln!();
        println_error(&format!("{err}"));
```

Reply:
I tried this at first and for some reason it doesn't have the desired effect with the ANSI `\x1b[2K` character that replaces the previous line.

Reply:
interesting; if both approaches fail then feel free to leave as is; I assumed it was low-effort fix